### ISSUE TITLE
Adding an invalid import causes an "IndexOutOfRangeException"

### ISSUE THREAD

**Issue by davkean** [dotnet/project-system#1985]
Note: Legacy project system

1. File -> New Console Application (.NET Framework)
2. Project Properties -> References
3. Imported Namespcae, add "Foo Bar"
4. Click Add User Import

```
---------------------------
Microsoft Visual Studio
---------------------------
Index was outside the bounds of the array.
---------------------------
OK   
---------------------------

```

**Comment by davkean** [comment]
I suspect this was regressed in Dev14.

**Comment by davkean** [comment]
Once you've done this - you can't close the window and you need to tear down VS.

**Comment by Pilchie** [comment]
Fixed by #3095
